packages/daos/mongo/src/index.ts

```diff
@@ -1,10 +1,28 @@
 import MongoBase from './base-model';
 import mongoDao from './dao';
 import { ObjectId } from 'mongodb';
+import * as yup from 'yup';
 import { DaoPackage, JsonDaos, JsonEnvs, EnvFile } from '@restlessness/utilities';
 
 import { modelTemplate } from './templates';
 
+class ObjectIdSchema extends yup.mixed {
+  constructor() {
+    super({ type: 'objectId' });
+
+    // @ts-ignore
+    this.withMutation(schema => {
+      schema.transform(function(value) {
```

Comment:
maybe use arrow function?

Reply:
I'm not sure if it was made on purpose to avoid scope issue with "this"